packages/common/src/services/integrations/jwp/JWPCheckoutService.ts

```diff
@@ -99,9 +99,12 @@ export default class JWPCheckoutService extends CheckoutService {
       customerCurrency: offer.currency,
       offerTitle: offer.description,
       active: true,
-      period: offer.access_type.period === 'month' && offer.access_type.quantity === 12 ? 'year' : offer.access_type.period,
-      freePeriods: offer.trial_period ? 1 : 0,
+      period:
+        offer.access_type.period === 'month' && offer.access_type.quantity === 12 ? 'year' : (offer.access_type.period as 'day' | 'week' | 'month' | 'year'),
```

Comment:
Is it possible that it is more than a year?

Reply:
If it's something like 13 months we'll display "13 months".